apprentice_teacher.rb

```diff
@@ -1,21 +1,17 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'school'
+require_relative 'teachable'
+
+
+class ApprenticeTeacher < School
+  include Teachable
+
+  attr_reader :salary, :target_raise, :rating_criteria
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super(age: 0, name: "")
     @target_raise = 800
     @phase = 3
-  end
-
-  def offer_high_five
-    "High five!"
-  end
-
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
+    @rating_criteria = 80
```

Comment:
This works, but I'd lean toward a constant for this, since it's fixed for all instances of ApprenticeTeacher.